unlock-app/src/utils/unlockUrl.js

```diff
@@ -0,0 +1,20 @@
+const unlockUrl = (env, config) => {
+  if (env && env.UNLOCK_URL) return env.UNLOCK_URL
```

Comment:
I now understand the thing above better. Would you rather pass `env.UNLOCK_URL` as a variable named `default` maybe? I think it's better if we try to isolate the use of `env`.

Reply:
Also I am a bit confused. It looks like this is duplicating the logic in `config.js` ?

Reply:
I didn't see any comparable logic in `config.js`, but happy to be corrected. I originally had it there but had feedback that I should move it into next.config.js, so that's what I did :)

The reason why I passed the whole logic here is so that I could test it - unfortunately I can't write tests for `next.config.js` itself. But yeah, I'll refactor to pass the env property differently.

Reply:
HA! I meant the logic below: setting the URL based on the env (staging, dev or prod...)